com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/message/Messages.java

```diff
@@ -108,14 +108,14 @@
     public static final String ERROR_DETERMINING_ACTIONS_TO_EXECUTE_ON_APP = "Error determining actions to execute on application \"{0}\"";
     public static final String ERROR_STOPPING_APP = "Error stopping application \"{0}\"";
     public static final String ERROR_STAGING_APP_1 = "Error staging application \"{0}\"";
-    public static final String ERROR_STAGING_APP_2 = "Error staging application \"{0}\": {1}";
+    public static final String ERROR_STAGING_APP_2 = "Staging of application \"{0}\" has failed. Check the application logs downloaded from the dmol command";
     public static final String ERROR_PUBLISHING_PUBLIC_PROVIDED_DEPENDENCIES = "Error publishing public provided dependecies";
     public static final String ERROR_STARTING_APP_1 = "Error starting application \"{0}\"";
-    public static final String ERROR_STARTING_APP_2 = "Error starting application \"{0}\": {1}";
+    public static final String ERROR_STARTING_APP_2 = "Application \"{0}\" startup has failed. Check the application logs downloaded from the dmol command";

```

Comment:
Change it to `"Startup of application \"{0}\" failed..."`. That way it would be more consistent with the error message from above.

Also, I think that when an application crashes, we also print the messages stored in the constants `SOME_INSTANCES_ARE_FLAPPING`/`SOME_INSTANCES_HAVE_CRASHED`. They also advise the user to `"Check the logs of [his] application for more information."` and if that's really the case, then there's a duplication of suggestions there.